### PR TITLE
Displayed predictions bugfix.

### PR DESCRIPTION
**Pull request by miskosz**

### CODE REVIEW

**Comment by miskosz** [comment]
Wait, I forgot one file

File: python/covid_web/covid_web/country_dashboard.py [hunk 1]
```diff
@@ -313,8 +311,7 @@ def _get_header_content(title: str) -> List[Component]:
         ),
         html.Ul(
             children=[
-                html.Li("Solid line is prediction"),
-                html.Li("Dashed line marks the culmination of the prediction"),
+                html.Li("Star marks the peak of the prediction"),
```
Comment (lukipuki) [inline python/covid_web/covid_web/country_dashboard.py]: Please revert this change, you are creating an unnecessary conflict for something I've already fixed it in an earlier PR: https://github.com/lukipuki/COVID-19-simulation/pull/104/files#diff-7ecadb1dbbbb3e136b5d83ad6bc8a7f1R320-R321

**Review by lukipuki — APPROVED** [review]
